Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/HibernateAccessor.java

```diff
@@ -58,8 +58,9 @@ public static HibernateAccessor getInstance() {
     public synchronized File getHibernateFile(String hibernateFileName) throws PropertyAccessException {
         checkEnvVarSet();
         LOG.debug("Checking the hibernate file.");
-        LOG.debug(propertyFileDir + File.separator + "hibernate" + File.separator + hibernateFileName);
-        File result = new File(propertyFileDir + File.separator + "hibernate" + File.separator + hibernateFileName);
+        String path = propertyFileDir + File.separator + "hibernate" + File.separator + hibernateFileName;
```

Comment:
Recommend to use StringBuilder to concatenate string

Reply:
Corrected to use StringBuilder.